force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +105,18 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+
+    public static Map<Id,Task> returnTasks() {
+        //  The body of the method
+        List<Task> taskList = [SELECT Id, ActivityDate FROM Task WHERE ActivityDate = Next_N_Days:7];
```

Comment:
Great use of these key words like NEXT_N_DAYS:n but remember that key words in queries are capitalized.